content/packages-and-modules/contributing-packages-to-the-registry/creating-and-publishing-scoped-public-packages.mdx

```diff
@@ -101,7 +101,7 @@ By default, scoped packages are published with private visibility. To publish a
 For more information on the `publish` command, see the [CLI documentation][cli-publish].
 
 
-[scopes]: about-scopes
+[scopes]: https://docs.npmjs.com/about-scopes
```

Comment:
Doesn't it only require the leading `/`?

Reply:
```sugestion
[scopes]: /about-scopes
```
indeed